src/style/index.css

```
@@ -365,7 +365,8 @@
     }
 
     /* Collapse everything else for single-print (only when multi-print is NOT also active) */
-    body:has(#single-print):not(:has(#multi-print)) *:not(#single-print):not(#single-print *) {
+    body:has(#single-print):not(:has(#multi-print))
+      *:not(#single-print):not(#single-print *) {
```

Stylelint (17.6.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)